api/src/application/user/user-emails.js

```diff
@@ -131,6 +131,10 @@ module.exports = (emailService, themeService, renderTemplate, emailTokenService)
       const token = await emailTokenService.create(user.get('id'), hoursTilExpiration, saveOptions);
       const viewContext = userViews.inviteEmail(user, client, config('/baseUrl'), {...query, token: token.get('token')});
 
+      if(query.subject) {
+        viewContext.subject = query.subject;
```

Comment:
Small update - instead of overriding the `viewContext.subject` here you can do it in the `inviteEmail()` view. If you check out that function it's pretty straight forward and it's already taking the whole destructured query object